test/e2e/node/kubelet.go

```diff
@@ -658,6 +661,74 @@ var _ = SIGDescribe("kubelet", func() {
 	})
 })
 
+var _ = SIGDescribe("specific log stream", framework.WithFeatureGate(features.SplitStdoutAndStderr), func() {
```

Comment:
This test should not run in the default jobs. It would be better to use `WithFeature` with adding the feature to [`ValidFeatures`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/feature/feature.go) like other features:
```suggestion
var _ = SIGDescribe("specific log stream", feature.SplitStdoutAndStderr), func() {
```

Reply:
Thanks for the pointer! I have updated the test